src/main/resources/schemas/iac_daily_list.json

```diff
@@ -212,20 +183,6 @@
                           "sittings"
                         ],
                         "properties": {
-                          "sessionChannel": {
```

Comment:
Changes look good but this field will need to be removed from the confluence page too. Also the example JSON on that confluence page will need to have those elements removed :) 
https://tools.hmcts.net/confluence/display/PUBH/Immigration+and+Asylum+Daily+Cause+List